Laura/index.js

```diff
@@ -0,0 +1,40 @@
+// # 6-PB-operators-arithmetic-pt2
+// # Programming Basics: Arithmetic Operators (Math Problems)
+
+// // **Basic Arithmetic Problems**
+
+// Q1. In one night, a movie theater sells tickets for 6450 dollars. Each ticket costs 15 dollars. How many tickets were sold?
+let priceTotal = 6450;
+let priceTicket = 15;
+console.log(priceTotal / priceTicket); // 430
+
+// Q2. Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let weekIncome = 500;
+let weeksPerMonth = 4;
```

Comment:
The second answer is wrong. There is 52 weeks per year and not 48!
The solution is :
 500 * 52 which the result is 26k. But your result is 24k a year!